.github/actions/fetch_ctk/action.yml

```diff
@@ -20,35 +20,11 @@ runs:
         echo "CTK_CACHE_FILENAME=mini-ctk-${{ inputs.cuda-version }}-${{ inputs.host-platform }}.tar.gz" >> $GITHUB_ENV
 
     - name: Install dependencies
-      shell: bash --noprofile --norc -xeuo pipefail {0}
-      run: |
-        dependencies=(zstd curl xz-utils)
-        dependent_exes=(zstd curl xz)
-
-        not_found=0
-        for dep in ${dependent_exes[@]}; do
-          if ! (command -v curl 2>&1 >/dev/null); then
```

Comment:
FYI, during refactoring I noticed this bug (`curl` was hard-wired here). It's fixed now.